react/react-training/src/redux/login/actions.js

```diff
@@ -1,29 +1,135 @@
 import UserService from '@services/UserService.js';
-import { ERROR_READING_RESPONSE, MSG_UNKNOWN_ID } from '@screens/Login/validation';
+import { ERROR_READING_RESPONSE, MSG_UNKNOWN_ID, MSG_UNKNOWN_USER } from '@screens/Login/validation';
 
 export const actionTypes = {
-  LOGIN: 'LOGIN',
-  LOGIN_FAILURE: 'LOGIN_FAILURE',
-  LOGIN_SUCCESS: 'LOGIN_SUCCESS',
-  LOGIN_VERIFY: 'LOGIN_VERIFY',
-  LOGOUT: 'LOGOUT'
+  // Get User Service
+  USER_GET: 'USER_GET',
+  USER_GET_FAILURE: 'USER_GET_FAILURE',
+  USER_GET_SUCCESS: 'USER_GET_SUCCESS',
+
+  // Login Service
+  USER_LOGIN: 'USER_LOGIN',
+  USER_LOGIN_FAILURE: 'USER_LOGIN_FAILURE',
+  USER_LOGIN_SUCCESS: 'USER_LOGIN_SUCCESS',
+
+  // Patch User Service
+  USER_PATCH: 'USER_PATCH',
+  USER_PATCH_FAILURE: 'USER_PATCH_FAILURE',
+  USER_PATCH_SUCCESS: 'USER_PATCH_SUCCESS',
+
+  // Post User Service
+  USER_POST: 'USER_GET',
+  USER_POST_FAILURE: 'USER_GET_FAILURE',
+  USER_POST_SUCCESS: 'USER_GET_SUCCESS',
+
+  // Other actions
+  USER_LOGIN_VERIFY: 'USER_LOGIN_VERIFY',
+  USER_LOGOUT: 'USER_LOGOUT'
 };
 
 export const actionsCreators = {
-  login: (username, password) => async dispatch => {
-    dispatch({ type: actionTypes.LOGIN });
+  userGet: userId => async dispatch => {
```

Comment:
I think this action shouldn't go here because this isn't associated with `Login` component